website/docs/r/policy_set.html.markdown

```diff
@@ -77,6 +79,10 @@ The following arguments are supported:
 * `global` - (Optional) Whether or not policies in this set will apply to
   all workspaces. Defaults to `false`. This value _must not_ be provided if
   `workspace_ids` is provided.
+* `kind` - (Optional) The policy-as-code framework associated with the policy.
```

Comment:
The API returns an error when you apply policy_ids of a different `kind` than you specify on the set. I wonder if it would be worth calling out that this needs to match the kind of the underlying policies.